src/ResourceManager/Storage/Commands.Management.Storage/Commands.Management.Storage.csproj

```diff
@@ -60,8 +61,8 @@
     <Reference Include="Microsoft.Data.Services.Client, Version=5.8.1.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
       <HintPath>..\..\..\packages\Microsoft.Data.Services.Client.5.8.2\lib\net40\Microsoft.Data.Services.Client.dll</HintPath>
     </Reference>
-    <Reference Include="Microsoft.WindowsAzure.Storage, Version=9.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
-      <HintPath>..\..\..\packages\WindowsAzure.Storage.9.0.0\lib\net45\Microsoft.WindowsAzure.Storage.dll</HintPath>
+    <Reference Include="Microsoft.WindowsAzure.Storage, Version=9.1.0.0, Culture=neutral, processorArchitecture=MSIL">
```

Comment:
You will need to make the identical change in Commands.Storage.Netcore.csproj in this directory

Reply:
It seems "Commands.Management.Storage.Netcore.csproj" don't reference the changed dll (XSCL) currently. Should I add it?